src/Document/Metadata/Metadata.php

```diff
@@ -167,4 +167,13 @@ private static function resolveName(\ReflectionProperty|\ReflectionMethod $refle
 
         return null;
     }
+
+    private function getFacetType(\ReflectionProperty|\ReflectionMethod $attributesAware): string
```

Comment:
The other methods are static for no particular reason. Just pointing it out :D I think I read sometime in some SF code that static methods was faster or had a smaller memory footprint. I have never googled it or fact checked it :D